network-api/networkapi/wagtailcustomization/static/wagtailadmin/js/copy-helper.js

```diff
@@ -0,0 +1,40 @@
+// Which locales are we dealing with?
+const main = wagtailModelTranslations.defaultLanguage;
+const languages = wagtailModelTranslations.languages.map((v) =>
+  v.replace(`-`, `_`)
+);
+
+// Which fields are going to be potential deal-breakers?
+const fields = Object.fromEntries(
+  Array.from(
+    document.querySelectorAll(`.content input[type=text]`)
+  ).map((e) => [e.getAttribute(`name`), e])
+);
+
+// 1: Give staff a button that lets them reveal all locale fields, so they can
+// manually edit any field to be any value they want.
+const toggle = document.querySelector(`.locale.helper.button`);
+
+toggle.addEventListener(`click`, () => {
+  document.querySelectorAll(`.locale-picker .locale-toggle`).forEach((btn) => {
```

Comment:
could cache this selector, but there just aren't any performance bottlenecks on this page.